src/ape_ethereum/ecosystem.py

```diff
@@ -379,6 +396,37 @@ def decode_returndata(self, abi: MethodABI, raw_data: bytes) -> Tuple[Any, ...]:
 
         return tuple(output_values)
 
+    def _enrich_value(self, value: Any) -> Any:
```

Comment:
Some food for thought: IPython supports rich display through `_repr_pretty_`

Reply:
I implemented it and did some refactoring that was necessary that